src/main/kotlin/com/chattriggers/ctjs/internal/listeners/ClientListener.kt

```diff
@@ -66,12 +66,16 @@ object ClientListener : Initializer {
                 }
             }
 
-            if (World.isLoaded()) {
+            //#if MC>=12004
+            if (World.toMC()?.tickManager?.shouldTick() == true) {
+            //#endif
```

Comment:
Needs the else block, or do we only need to check this on 1.20+?

Reply:
Yeah tick manager is 1.20.3+ only, there is no concept of "not ticking" in previous versions

Reply:
Wouldn't you want to keep `World.isLoaded` for 1.19.4?

Reply:
Oh I didn't realize I removed that, yeah definitely

Reply:
Ahh actually it was fine the way it is, if `World.isLoaded()` is false, then `World.toMC()` would be null and the condition wouldn't be true